packages/js/data/src/items/reducer.ts

```
@@ -41,18 +41,22 @@ const reducer: Reducer< ItemsState, Action > = (
 			};
 		case TYPES.SET_ITEMS:
 			const ids: Array< ItemID > = [];
+			const resourceName = getResourceName(
+				action.itemType,
+				action.query
+			);
```

Biome (2.5.7)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)